config/s6-overlay/scripts/entry.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+# Inject number of processors into environment for confd.
+# This value is used by haproxy and openvpn config templates.
+NPROC=$(nproc)
+export NPROC
```

Comment:
This doesn't seem to be getting picked up by confd when run again as supervised which is odd, will need to do more debugging.